index.js

```diff
@@ -127,3 +127,38 @@ function closeWinfun()
   document.getElementById('cover').style.cssText = 'display: none'; 
   document.getElementById('body').style.cssText = 'overflow: none;';
 };
+
+
+
+// const email =document.querySelector('email').value;
+ 
+// form.addEventListener('submit', (e) =>{
+//   e.preventDefault();
+//   console.log("submit clicked!");
+// if(email != email.toLowerCase())
+// {
+// errorMsg =innerHTML= msg;
+// console.log("Error Msg show");
+
+// }
+// else{
+//   console.log("prevent def called!");
+//   console.log("email is "+email);
+// }
+//   }
+// );
```

Comment:
- [ ] (OPTIONAL) Kindly remove all commented code in order to keep your work clean 👍